modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java

```diff
@@ -466,5 +531,9 @@ public DeploymentService getDeploymentService() {
   public ArtifactClassLoader getContainerClassLoader() {
     return artifactResourcesRegistry.getContainerClassLoader();
   }
+
+  private boolean isFipsEnabled() {
+    return FIPS_SECURITY_MODEL.equals(getProperty(MULE_SECURITY_SYSTEM_PROPERTY));
```

Comment:
Are we planning to have other security models apart from this? If we are not probably this should be only a boolean. If we are planning to do that, we should add something to retrieve what we have to do according to the string (though in this case it seems a bit of overengineering to do that). 
Are we sure we want the providers to be hardcoded? Probably yes, as we will only support BouncyCastle.

Reply:
For now we are going to support only BouncyCastle, but we had other  thoughts like exposing as properties in wrapper.conf